chromatic/rainbows/actions/remove_trends.py

```diff
@@ -0,0 +1,120 @@
+from ...imports import *
+
+
+def remove_trends(
```

Comment:
@Pat-Wachiraphan , after thinking about it a while, I think we shouldn't pretend that these functions magically know what is astrophysics vs what is systematics. Let's just keep the name general as `remove_trends`, but we'll know we can often use it as a quick way to remove a transit!